maps/map_files/BigRed/BigRed.dmm

```diff
@@ -2004,7 +2002,6 @@
 /obj/structure/pipes/standard/simple/hidden/green,
 /obj/structure/machinery/door/airlock/almayer/security/glass/colony{
 	dir = 1;
-	locked = 0;
```

Comment:
Any reason as to why these doors are no longer locked or ?

Reply:
Unsure why these were changed. I'll go through and fix them. Should only be a moment.